app/Formatters/Openinghours.php

```diff
@@ -104,9 +104,9 @@ protected function toHTML()
                     foreach ($schedule as $entry) {
                         $formattedSchedule .= "<div>$entry</div>";
                     }
-                } else {
-                    $formattedSchedule .= "<div>$schedule</div>";
-                }
+                    continue;
+                } 
+                $formattedSchedule .= "<div>$schedule</div>";
```

Comment:
Can we use a template file to render the html version?

Reply:
probably, but HTML output will be refactored in a different issue.